docs/operate/reference/advanced-modules/_index.md

```diff
@@ -46,6 +46,74 @@ Running {{< glossary_tooltip term_id="modular-resource" text="modular resources"
 
 However, if you are unable to use modular resources because you need to host `viam-server` on a non-Linux system or have an issue with compilation, you may need to [implement a custom component and register it on a server configured as a remote](/operate/reference/advanced-modules/custom-components-remotes/) on your machine.
 
+## Package and deploy using Docker
+
+In rare cases, you may need to package and deploy a module using Docker.
+Use cases for this include:
+
+- Your module has complex system dependencies that cannot be installed on a machine.
+- You have a large bundle and want to use layer caching to reduce the size of the download.
```

Comment:
The world "bundle" seems odd, but I can't think of a better word. If no one else can think of a better word either, leave as-is.

Reply:
I suppose that depends on what's meant by bundle? Does the layer caching help because an image with the same base container and possibly some layers can be used between different installed modules? That's the only way I can think this would be helpful.

Reply:
The people who love docker will talk your ear off about how useful layer caching is: the idea is that if you have a bunch of small layers that add specific things to your docker image, you can reuse these layers in different projects to get different images without needing to create each thing multiple times. This is an argument for docker use in general, not something Viam-specific.

I think the bundle is supposed to refer to the collection of layers. I don't know enough about docker to know if that's the standard term; it might be. Even if it's not, I think the meaning is clear by the end of the sentence, I just stumbled a little in the middle.

Reply:
@abe-winter I borrowed your words that were perhaps not intended for official documentation. Is there a word other than "bundle" you'd recommend here?

Reply:
maybe 'artifact'?

Reply:
Right so it's useful if you have a large image and you use some of the same layers already on the machine so that they'll already be present. So that is the use case. Otherwise there wouldn't be anything reducing the size of the download. 

```suggestion
- You use a large container image and some layers are already used by your machine which means layer caching can reduce the size of the download.
```